components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminService.java

```diff
@@ -80,8 +80,9 @@
 
         String tenantUser = MultitenantUtils.getTenantAwareUsername(loggedInUser);
         int tenantId = CarbonContext.getThreadLocalCarbonContext().getTenantId();
+        String userDomain = UserCoreUtil.extractDomainFromName(loggedInUser);
```

Comment:
User IdentityUtil.extractDomainFromName because this method doesn't return domain even if primary user store is given an explicit domain name.